ExadelTimeTrackingSystem/ExadelTimeTrackingSystem.BusinessLogic/Validators/BulkCreateTaskDTOValidator.cs

```diff
@@ -23,7 +23,8 @@ public BulkCreateTaskDTOValidator(IValidator<CreateTaskDTO> createTaskDtoValidat
         private void ConfigureRules()
         {
             RuleFor(t => t.Task)
-                .SetValidator(_createTaskDtoValidator);
+                .SetValidator(_createTaskDtoValidator)
+                .NotEmpty();
```

Comment:
logically we should check NotEmpty first before setting validator, otherwise .SetValidator can cause null reference exception. By the way have you checked if this combination SetValidator and some additional rules as NotEmpty, is working?